private/tools/prebuilt/BUILD

```diff
@@ -1,7 +1,9 @@
 # This package contains static executables used by the maven_install repository rule.
+# To update the prebuilt binaries, `bazel run //scripts:refresh-prebuilts`
 
 exports_files([
-    "hasher_deploy.jar",  # built from //private/tools/java:hasher_deploy.jar
-    "list_packages_deploy.jar",  # build from //private/tools/java/rules/jvm/external/jar:ListPackages_deploy.jar
+    "hasher_deploy.jar",  # built from //private/tools/java/rules/jvm/external:hasher-tool_deploy.jar
```

Comment:
Do you know if we can use buildkite to create these jars on each CI build? Then at least we'd have non-local dev builds.

Reply:
I'm not sure. We'd also (presumably) want buildkite to update the PR or fail the build if these change....

Reply:
Yeah that could work too. Mind adding a test?